docs.openc3.com/docusaurus-plugin.config.js

```diff
@@ -17,6 +17,10 @@ const config = {
   tagline: 'OpenC3 COSMOS Documentation',
   favicon: 'img/favicon.png',
 
+  future: {
+    experimental_faster: true,
+  },
```

Comment:
This makes the build faster?

Reply:
Yep.  See the Docusaurus 3.6 release notes.